tools/build-debian-copyright

```diff
@@ -114,11 +114,13 @@ def module_copyright(moddir):
     except subprocess.CalledProcessError:
         pass
 
-    # missing copyrights
-    if mod == 'kubernetes-object-describer' or mod == 'object-describer':
-        # only committer is Jessica Forrester, working for Red Hat
-        # https://github.com/kubernetes-ui/object-describer/issues/35
-        copyrights.add(' (C) 2015-2017 Red Hat, Inc.')
```

Comment:
Are we sure we want to remove the red hat one? Seems it's still valid for these 2 packages.

Reply:
This copyright statement was just a bold guess of mine - as the issue says, this is nowhere to be found in the source. This PR does not add an explicit "(C) <year>" any more, but just states the author. I think this is preferrable after all.

However, I don't have a strong opinion about this - if you prefer to keep this special case I can change accordingly.

Reply:
Ok, i'm pretty sure in this case it was correct. But I don't have a strong opinion on it.